conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
```

Comment:
conftest.py is a magic pytest file that is automatically loaded. it's a bit weird, but this is how you do it